pull_request_template.md

```diff
@@ -1,20 +1,18 @@
-If Applicable:
-Promotion to UAT:
-- LINK_TO_LOWER_ENVIRONMENT_PRS
+# Description
 
-----
+Please include a summary of the changes and the related issue.
 
-< list your changes here >
- 
--- Reminders (Remove this from template before submitting) --
+Fixes # (issue)
 
-Did you remember to:
-- **Set this PR's target branch!  It should probably be `puat` or `uat`, not `master`!**
-- Rebase your issue branch onto the target branch. If the commit list below includes anything other than your commits, you should rebase.
+## Type of change
 
-Consider the following about your changes:
-- Do your commit messages reference the ticket ID?
```

Comment:
I think we should keep this in the checklist.  Luckily this is mostly engrained in the culture but still worth reminding I think.